assets/js/delete-task.js

```diff
@@ -0,0 +1,9 @@
+import { CLICK_EVENT } from "/assets/js/constants.js";
+
+function deleteTaskEvent(deleteButton, task, taskList, allTasks, id) {
```

Comment:
function parameter `taskList` and `allTasks` are ambiguous and hard to differentiate their responsibilities. it is better and much clearer to use a suffix `El` or prefix `$` for variable names for the DOM nodes such as `$taskList`, `taskEl` etc. thus variable `tasks` becomes clear and represents the model.

```suggestion
function deleteTaskEvent(deleteButton, task, taskListEl, tasks, id) {
```